app/views/admin/pages/_slices.html.erb

```diff
@@ -13,12 +13,16 @@ $(function() {
       metaExtraTemplates: <%= raw @page.meta_extra_templates.to_json %>,
 
       super_user: <%= current_admin.super? %>,
+      locale: '<%= I18n.locale %>',
+
+      loadPagePath: '/admin/pages/{{id}}.json?locale=<%= I18n.locale %>',
+      savePagePath: '/admin/pages/{{id}}.json?locale=<%= I18n.locale %>',
```

Comment:
Ok yeah, I see what you mean. `mainTemplate` and `metaTemplate` are also defined here, so it seems like the only thing that is _ever_ taken from the other place is `templatesPath`. Am I right? If so it would be good to clean that up.